packages/ra-ui-materialui/src/input/AutocompleteInput.tsx

```diff
@@ -328,13 +332,6 @@ const AutocompleteInput: FunctionComponent<
                             name={input.name}
                             InputProps={{
                                 inputRef: storeInputRef,
-                                classes: {
-                                    root: classNames(classes.inputRoot, {
```

Comment:
this was used to support both filled and standard variants. I'm afraid this change breaks that. Can you make sure the form displays correctly with `<SimpleForm variant="standard">`?

Reply:
Confirmed. It works by setting the variant on `SimpleForm`